OpenAPI/api-contract.yaml

```diff
@@ -0,0 +1,136 @@
+openapi: 3.0.1
+info:
+  title: Maintenance API - Event Driven Internal
+  description: |
+    # Overview
+    Maintenance API - Event Driven Internal is an internal API used in the event driven solution for Maintenance API. 
+
+    The consumer facing documentation is available at https://equinor.github.io/maintenance-api-event-driven-docs/
+  version: "0.1.0"
```

Comment:
is this needed for internal tracking or oas syntax?

Reply:
Think the version is required by OAS. 
Leaving as 0.1.0 now and once we are ready for prod it will be moved to 1.0.0(and will likely stay stable for a long time). 

api-version is turned off in APIM for this API, so we won't have to deal with it